flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/java/windows.scala

```diff
@@ -127,3 +127,21 @@ class PartitionedOver(private val partitionByExpr: Array[Expression]) {
     new OverWindowWithOrderBy(partitionByExpr, orderByExpr)
   }
 }
+
+
+class OverWindowWithOrderBy(
+  private val partitionByExpr: Array[Expression],
```

Comment:
Suggest that change `partitionByExpr` to `partitionBy` for keep param name consistent with SCALA. What do you think?